pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -31,9 +29,16 @@ stdenv.mkDerivation rec {
     homepage = "http://rg3.github.com/youtube-dl/";
     repositories.git = https://github.com/rg3/youtube-dl.git;
     description = "Command-line tool to download videos from YouTube.com and other sites";
-    license = stdenv.lib.licenses.unlicense;
-
-    platforms = with stdenv.lib.platforms; linux ++ darwin;
-    maintainers = with stdenv.lib.maintainers; [ bluescreen303 simons phreedom AndersonTorres fuuzetsu ];
+    longDescription = ''
+      youtube-dl is a small command-line program to download videos
+      from YouTube.com and a few more sites. It requires the Python
```

Comment:
I understand that :)
But due to nixos reality, we don't need this sentence.

Sincerely,
Arseniy Seroka



On 5 April 2015 18:23:45 Anderson Torres <notifications@github.com> wrote:

> > @@ -31,9 +29,16 @@ stdenv.mkDerivation rec {
> >      homepage = "http://rg3.github.com/youtube-dl/";
> >      repositories.git = https://github.com/rg3/youtube-dl.git;
> >      description = "Command-line tool to download videos from YouTube.com 
> and other sites";
> > -    license = stdenv.lib.licenses.unlicense;
> > -
> > -    platforms = with stdenv.lib.platforms; linux ++ darwin;
> > -    maintainers = with stdenv.lib.maintainers; [ bluescreen303 simons 
> phreedom AndersonTorres fuuzetsu ];
> > +    longDescription = ''
> > +      youtube-dl is a small command-line program to download videos
> > +      from YouTube.com and a few more sites. It requires the Python
>
> Well, I just copied from official page.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/NixOS/nixpkgs/pull/7179/files#r27778036